src/backend/distributed/executor/repartition_logic.c

```diff
@@ -142,28 +138,7 @@ CreateSchemasOnAllWorkers(char *createSchemasCommand)
 {
```

Comment:
Maybe rename to SendCommandToAllWorkers.

Reply:
Rename what to `SendCommandToAllWorkers` ?

Reply:
CreateSchemasOnAllWorkers

Reply:
Umm, makes sense I will move this to worker_transaction.c